src/Tickets/Seating/app/blockEditor/hook-callbacks.js

```diff
@@ -153,11 +153,46 @@ export const filterSettingsFields = (fields) => {
 	const layouts = store.getLayoutsInOptionFormat();
 
 	fields.push(
-		<LayoutSelect
-			layouts={layouts}
-			currentLayout={currentLayout}
-		/>
+		<LayoutSelect layouts={layouts} currentLayout={currentLayout} />
 	);
 
 	return fields;
-}
\ No newline at end of file
+};
+
+/**
+ * Filters whether the ticket is valid.
+ *
+ * @since TBD
+ *
+ * @param {boolean} isValid  Whether the ticket is valid.
+ * @param {Object}  state    The state of the store.
+ * @param {Object}  ownProps The own props of the component.
+ *
+ * @return {boolean} Whether the ticket is valid.
+ */
+export const filterTicketIsValid = (isValid, state, ownProps) => {
```

Comment:
This function might likely require modification following my comment on the later application of the filter.